runtime/config.go

```diff
@@ -46,6 +47,10 @@ type Config struct {
 	LogLevel              string `json:"log_level"`
 	HtEnabled             bool   `json:"ht_enabled"`
 	Debug                 bool   `json:"debug"`
+
+	DisableJailing bool `json:"disable_jailing"`
```

Comment:
Similar to my above comment, I think it may be more flexible to make "JailingConfig" it's own struct within this one rather than put all the fields on the top level.